snuba/datasets/outcomes_processor.py

```diff
@@ -26,10 +26,11 @@ def process_message(
         assert isinstance(value, dict)
         v_uuid = value.get("event_id")
 
-        if "category" not in value:
-            metrics.increment("missing_category")
-        if "quantity" not in value:
-            metrics.increment("missing_quantity")
+        if value["outcome"] != 4:  # we dont care about abuse outcomes for these metrics
```

Comment:
Do we have a constant for this? 😬

Reply:
we don't import sentry in this service and the added work of copying over the constant didn't seem worth it, as we'll likely remove these metrics in a couple of weeks and it wouldn't be used anywhere else. that being said, i do think we could do something similar to `DataCategory` wherein the source of truth for that enum lives in sentry_relay and can be freely imported across our codebases. @jan-auer do you think this would be a good idea?

Reply:
Agreed, definitely makes sense to centralize the schema and definitions. It's still heavy to introduce enums like this, but we can improve the tooling around this.